libraries/lib-module-manager/AsyncPluginValidator.cpp

```diff
@@ -57,9 +57,8 @@ class AsyncPluginValidator::Impl final :
    {
       auto server = std::make_unique<IPCServer>(*this);
       if(!PluginHost::Start())
-         mDelegate->OnInternalError("Cannot start plugin host");
-      else
-         mServer = std::move(server);
+         throw std::runtime_error("cannot start plugin host process");
```

Comment:
This possibly thrown exception is not caught anywhere and might terminate the program.  One remedy would be, to move initialization of PluginStartupRegistration::mValidator out of the constructor, and into the try-catch block in Run().

Reply:
StartHost isn't called from the constructor